src/TTextEdit.cpp

```diff
@@ -498,14 +502,6 @@ void TTextEdit::drawLine(QPainter& painter, int lineNumber, int lineOfScreen) co
  */
 int TTextEdit::drawGrapheme(QPainter& painter, const QPoint& cursor, const QString& grapheme, int column, TChar& charStyle) const
 {
-    uint unicode = getGraphemeBaseCharacter(grapheme);
-    int charWidth;
-    if (unicode == '\t') {
```

Comment:
How well does the new code work with tab characters given that you are doing away with the current faking of (Left) Tabs...

Reply:
That's an open point I was working on.
The code you quoted was not symmetric to the code in `convertMouseXToBufferX`, which was:
`            if (unicode == '\t') {
                characterWidth = 8;
            } else {
                characterWidth = getGraphemeWidth(unicode);
            }
`
and it confused me. I'll fix it as soon as possible.